ui/components/multichain/pages/all-connections/all-connections.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
```

Comment:
Can we have all new files in tsx to reduce the future work for typescript migration :)

Reply:
[Will be addressed in future PRs](https://github.com/MetaMask/metamask-extension/pull/21332#discussion_r1367142422)